src/components/ItemInfo/ItemInfo.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  useDisclosure,
+  Button,
+  Heading,
+  UnorderedList,
+  ListItem,
+} from '@chakra-ui/react';
+
+const ItemInfo = ({ items, isAccepted = false }) => {
```

Comment:
🚫 **[eslint]** <[react/prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'items' is missing in props validation